docs/blog/posts/pydantic.md

```diff
@@ -0,0 +1,235 @@
+---
+draft: False
+date: 2023-12-11
+slug: pydantic
+tags:
+  - pydantic
+authors:
+  - jxnl
+---
+
+# Steering Large Language Models with Pydantic
+
+In the past year, significant progress has been made in utilizing large language models. Prompt engineering, in particular, has gained attention, and new prompting techniques are being developed to guide language models toward specific tasks. While many are building chat bots, an even more exciting application is the generation of structured outputs, whether its extracting structured data, augmenting your RAG application, or even generating
+
+While some have resorted to [threatening human life](https://twitter.com/goodside/status/1657396491676164096?s=20) to generate structured data, we have found that Pydantic even more effective.
```

Comment:
Haha, funny example.

```suggestion
While some have resorted to [threatening human life](https://twitter.com/goodside/status/1657396491676164096?s=20) to generate structured data, we have found that Pydantic is even more effective.
```

Reply:
Ok, after reading further along in your blog, this example is even more relevant than I thought. Perhaps a bit more contextualization of the example could help to set the stage a bit better for the patterns to come in the post.

I think having an out-there, extreme example like this is both funny and helpful, and would be even more useful than it already is with just a bit more context.

Reply:
@jxnl, I think you still need the "Pydantic _is_"